client/src/components/InlinePanel/index.js

```diff
@@ -209,6 +209,28 @@ export class InlinePanel extends ExpandingFormset {
     );
   }
 
+  addTabIndex(prefix) {
+    /*
+    add tabindex -1 into newly created form and remove it from old forms
+    */
+    const forms = this.formsElt.children(':not(.deleted)');
+    const inlineChildId = 'inline_child_' + prefix;
+    const formsArray = forms.toArray();
+
+    formsArray.forEach(function(node) {
+      const $node = $(node);
+
+      if ($node.attr('id') === inlineChildId) {
+        $node.attr('tabindex', -1);
+        $node.focus(function() {});
+        $node[0].scrollIntoView();
```

Comment:
Drive by comment. I have this run locally. 

However, this will need to call focus on the element, maybe with smooth scroll. 

Then it'll need to add an event listener for blur to remove the tab index. But only if it wasn't already there in the first place. 

Otherwise the tabIndex will stay when not required. 

Finally, below when you remove the tabIndex, that's probably not required - we should leave attributes as is if the node is not the one with the Id.

Reply:
Thanks for your suggestions @lb-  since its my first contribution into wagatil